test/unit_tests/braket/circuits/test_angled_gate.py

```diff
@@ -61,6 +61,26 @@ def test_equality(angled_gate):
     assert angled_gate != non_gate
 
 
+def test_symbolic_equality():
+    symbol1 = FreeParameter("theta")
+    symbol2 = FreeParameter("phi")
+    gate1 = AngledGate(angle=symbol1, qubit_count=1, ascii_symbols=["bar"])
+    gate2 = AngledGate(angle=symbol1, qubit_count=1, ascii_symbols=["foo"])
+    other_gate = AngledGate(angle=symbol2, qubit_count=1, ascii_symbols=["foo"])
+
+    assert gate1 == gate2
```

Comment:
Can you add tests/assertions where the FreeParameter objects are different, but their names are the same? And tests/assertions for what happens when you fix parameters (with same/different values).

Reply:
Added in different equality checks in the free parameter checks